Update XMCL install/export help (#47)
modrinth/docs: docs/modpacks/creating_modpacks.md

When you have a working packwiz pack, you can use the [`packwiz mr export`] command to create a Modrinth modpack.

[`packwiz mr export`]: https://packwiz.infra.link/reference/commands/packwiz_modrinth_export/

### X Minecraft Launcher (XMCL)

> X Minecraft Launcher (XMCL) is an open source Minecraft launcher provides a disk efficient way to manage massive mods & other resources via instances model similar to MultiMC.

Reviewer: ```suggestion
X Minecraft Launcher (XMCL) is an open source Minecraft launcher which provides a disk efficient way to manage massive mods and other resources similarly to MultiMC.
```

You can enable/disable the mods in `Mods` section.

![Mods](../../static/img/tutorial/xmcl-mods.png)

In home page, click the `Export as Modpack` button to open the export dialog.

![Export](../../static/img/tutorial/xmcl-export.png)

> You can config your modpack in this dialog, like setting the client/server side setting for mods files.

Reviewer: ```suggestion
You can configure your modpack in this dialog -- for example, you can the environment (client-side/server-side) for each file.
```

> Once you done, click the `EXPORT AS MODPACK` button to export `.mrpack` format modpack.

Reviewer: ```suggestion
Once you done, click the `Export as Modpack` button to export to the Modrinth format.
```

![Export](../../static/img/tutorial/xmcl-export-dialog.png)

> 
> :::tip
> Since Curseforge now does not allow Modrinth modpack includes curseforge CDN url, the files downloaded from Curseforge will not be able to list in `modrinth.index.json` with url. It will be compressed as a file in overrides.
> 
> You need to download the file from Modrinth if you want to use them as url.
> :::

Reviewer: ```suggestion
```

:::caution

> Since XMCL currently does not support download mods from Github/Gitlab, setting file url from github/gitlab is unsupported. This will be supported in later feature update.

Reviewer: ```suggestion
XMCL currently does not support download mods from GitHub or GitLab. File URLs from these sites are unsupported.
```
